js_modules/dagit/packages/core/src/workspace/WorkspaceOverviewWithGrid.tsx

```diff
@@ -22,62 +22,62 @@ import {repoAddressAsString} from './repoAddressAsString';
 import {workspacePath} from './workspacePath';
 
 export const WorkspaceOverviewWithGrid = () => {
+  return (
```

Comment:
Hmm, I wonder if it'd make sense to start calling things like this `WorkspaceOverviewWithGridOSS`  to make it more clear that there's an alternative version in cloud... I worry this is the sort of thing I will see in 6mo and think "this abstraction doesn't really seem necessary"  and fold the two components back together next time I need to do something structural to the page.

I wish we could have a "oss export" keyword or a `public` annotation or something :(  Not sure any such thing exists but I'd be nice if renaming or removing WorkspaceOverviewGrid would break the build locally.

Nothing immediately actionable though - will have to think about it a bit! :-/

Reply:
Yeah, at least in this case if it were refactored away, it would break Cloud TS. But we do need to be careful to test Cloud when making changes to shared/forked parts of the app.

Reply:
Yeah I think that's true... just wish there was some indication of what was a `shared/forked part of the app`.  👍